src/lsptoolshost/roslynProtocol.ts

```diff
@@ -260,3 +267,9 @@ export namespace RestorableProjects {
     export const messageDirection: lsp.MessageDirection = lsp.MessageDirection.clientToServer;
     export const type = new lsp.RequestType0<string[], void>(method);
 }
+
+export namespace ProjectHasUnresolvedDependenciesNotification {
```

Comment:
```suggestion
export namespace ProjectNeedsRestoreRequest {
```

Reply:
"Request" seems most critical if the API pattern distinguishes request vs. notification to avoid other confusion, but the rest seems good too?

Reply:
changed to request.  if I'm changing the underlying method name, it will need a server update.  I'll do all that separately.